docs/en/end-user/traits/more.md

```diff
@@ -17,7 +17,7 @@ For example, let's try to list all available traits in registry:
 
 ```shell
 $ kubectl vela trait --discover
-Showing traits from registry: https://github.com/oam-dev/catalog/tree/master/registry
+Showing traits from registry: https://kubevela-registry.oss-cn-beijing.aliyuncs.com
```

Comment:
let's use this domain:  registry.kubevela.net

Reply:
@zzxwill

Reply:
done